# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/cdrdao/make-more.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- makefile.org	Sat Dec 28 23:21:18 2002
+++ makefile	Sat Dec 28 23:21:31 2002
@@ -50,10 +50,10 @@
 	@echo antlr executable now in $(BINDIR)
 	(cd ./dlg; $(MAKE) CC="$(CC)" COPT="$(COPT)")
 	@echo dlg executable now in $(BINDIR)
-#	(cd ./sorcerer; $(MAKE) CC="$(CC)" COPT="$(COPT)")
-#	@echo sorcerer executable now in $(BINDIR)
-#	(cd ./support/genmk; $(MAKE) CC="$(CC)" COPT="$(COPT)"; mv genmk ../../$(BINDIR))
-#	@echo genmk executable now in $(BINDIR)
+	(cd ./sorcerer; $(MAKE) CC="$(CC)" COPT="$(COPT)")
+	@echo sorcerer executable now in $(BINDIR)
+	(cd ./support/genmk; $(MAKE) CC="$(CC)" COPT="$(COPT)"; mv genmk ../../$(BINDIR))
+	@echo genmk executable now in $(BINDIR)
 	@echo
 	@echo "       PCCTS 1.33MR32 installation complete"  # MRXXX
 
